BuildResidentialHPXML/measure.rb

```diff
@@ -3648,14 +3648,14 @@ def run(model, runner, user_arguments)
     # Create HPXML file
     hpxml_path = args[:hpxml_path]
     unless (Pathname.new hpxml_path).absolute?
-      hpxml_path = File.expand_path(hpxml_path)
+      hpxml_path = File.join(runner.workflow.absoluteRootDir.to_s, hpxml_path)
```

Comment:
[runner.workflow.absoluteRootDir: this will resolve to '.'. '.' is the location defined as the root key inside the workflow.osw, or if not specified the location of the workflow.osw itself](https://nrel.github.io/OpenStudio-user-documentation/reference/measure_writing_guide/#using-files-and-using-externalfile-in-a-measure)